app/javascript/application.js

```diff
@@ -0,0 +1,5 @@
+// Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
+import Rails from "@rails/ujs"
+import "channels"
+
+Rails.start()
```

Comment:
Entry point is moved from `app/javascript/packs/application.js` to `app/javascript/application.js`.
When using Webpacker, the entry point is under packs.